src/OpenTK/Platform/MacOS/Cocoa/NSFloat.cs

```diff
@@ -179,5 +191,17 @@ struct NSRect
             return new RectangleF(s.Location, s.Size);
         } 
     }
-}
 
+    // Using IntPtr in NSFloat cause that if imported function
+    // return struct that consist of them you will get wrong data 
+    // This types are used for such function.
```

Comment:
@mellinoe - Ah, you're 100% right as expected here. I've always seen it explicitly specified for interop types. We should probably add it to ditch the warning as you suggest, though.